docker/docker-compose.yml

```diff
@@ -0,0 +1,29 @@
+version: "3.3"
+
+services:
+
+  alephium:
+    image: alephium/alephium:latest
+    build:
+      context: .
+      args:
+        RELEASE: "0.7.3"
```

Comment:
Is there a way to share the release version across "docker-compose.yml" and "Dockerfile" ?

Reply:
The version used is the one in docker-compose. There is no need to update in Dockerfile.

Reply:
Updated in the latest commit